src/components/TimeseriesChartMeta/components/TimeseriesValue.tsx

```diff
@@ -71,18 +78,27 @@ export class TimeseriesValue
 
   getLatestDatapoint = async () => {
     try {
-      const datapoint = await connectPromiseToUnmountState(
+      const datapoints = await connectPromiseToUnmountState(
         this,
-        sdk.Datapoints.retrieveLatest(this.props.timeseriesName)
+        // @ts-ignore
+        this.context.datapoints.retrieveLatest([
+          { externalId: this.props.timeseriesName, before: 'now' },
```

Comment:
Can we use `timeresies.id` instead of `name`?
```suggestion
          { id: this.props.timeseriesId, before: 'now' },
```

Reply:
Fixed